src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Common/IProjectContextInfoExtensions.cs

```diff
@@ -68,6 +69,31 @@ public static class IProjectContextInfoExtensions
             }
         }
 
+        public static async ValueTask<IReadOnlyCollection<string>> GetPackageFoldersAsync(
```

Comment:
Could you add at least 1 line of details on what kind of package folder it returns?
Someone looking at this code later need to spend time to understand what it's.

Reply:
Added docs.